src/backend/settings.py

```diff
@@ -114,9 +118,9 @@
 
 MIDDLEWARE = [
     'django.middleware.security.SecurityMiddleware',
+    "corsheaders.middleware.CorsMiddleware",
```

Comment:
Why change the order? : D

Reply:
According to package official document https://pypi.org/project/django-cors-headers/

> CorsMiddleware should be placed as high as possible, especially before any middleware that can generate responses such as Django’s CommonMiddleware or Whitenoise’s WhiteNoiseMiddleware. If it is not before, it will not be able to add the CORS headers to these responses.

So, I arrange its order. ;)